Gemfile

```diff
@@ -37,6 +37,7 @@ gem 'bootsnap', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 gem 'active_model_serializers'
+gem 'iri'
```

Comment:
The gem version still missing.

Reply:
907d9d7bf1a48d657192fca59beabe51170af9a8